ground/src/main/java/com/google/android/ground/ui/theme/Theme.kt

```diff
@@ -87,6 +91,8 @@ private val DarkColors =
     scrim = md_theme_dark_scrim,
   )
 
+val manropeFont = FontFamily(Font(R.font.manrope_medium, FontWeight.Normal))
```

Comment:
This can be cleaned up after addressing the other comment

Reply:
fixed